app/containers/RoomItem/index.tsx

```diff
@@ -124,7 +114,8 @@ const RoomItemContainer = React.memo(
 				autoJoin={autoJoin}
 				showAvatar={showAvatar}
 				displayMode={displayMode}
-				sourceType={item.source}
+				status={item.t === 'l' ? item?.visitor?.status : null}
```

Comment:
```suggestion
				status={item.t === SubscriptionType.OMNICHANNEL ? item?.visitor?.status : null}
```
Even though the type is wrong, it's better to use the type than a string.